examples/aws/verify_egressv1.go

```diff
@@ -0,0 +1,49 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
+	"github.com/openshift/osd-network-verifier/pkg/cloudclient"
+)
+
+// Example config struct
+type egressConfigV1 struct {
```

Comment:
would you like to remove this as well? (`egressConfigV1`) as it's now not being used

Reply:
thanks, removed